lib/shopify_api/rest/draft_order.ex

```diff
@@ -0,0 +1,14 @@
+defmodule ShopifyAPI.REST.DraftOrder do
+  @moduledoc """
+  """
+  alias ShopifyAPI.AuthToken
+  alias ShopifyAPI.REST.Request
+
+  @doc """
+    Create a new draft order
+
+  iex> ShopifyAPI.REST.DraftOrder.create(auth, %Order{})
```

Comment:
There's a few styles of method documentation used in this library, I'd prefer if we started using something closer to other libraries, more in this style:

```elixir
@doc """
Creates a new DraftOrder with the provided attributes.

## Examples

    iex> ShopifyAPI.REST.DraftOrder.create(auth_token, %{draft_order: %{}})
    {:ok, %{"draft_order" => %{...}}}
"""